src/compiler/crystal/compiler.cr

```diff
@@ -240,7 +240,7 @@ module Crystal
 
     private def codegen_single_unit(unit, target_triple, multithreaded)
       unit.llvm_mod.target = target_triple
-      ifdef x86_64
+      if target_triple =~ /^x86_64/
```

Comment:
Flags are now set from the target triple (default or specified). I suppose this is no longer required?